docs/fsharp/tutorials/type-providers/accessing-a-web-service.md

```diff
@@ -79,7 +79,7 @@ type TerraService = WsdlService<" HYPERLINK "http://terraserver-usa.com/TerraSer
 ```
 
   A red squiggle appears under this line of code if the service URI is misspelled or if the service itself is down or isn’t performing. If you point to the code, an error message describes the problem. You can find the same information in the **Error List** window or in the **Output Window** after you build.
-<br />  There are two ways to specify configuration settings for a WSDL connection, by using the app.config file for the project, or by using the static type parameters in the type provider declaration. You can use svcutil.exe to generate appropriate configuration file elements. For more information about using svcutil.exe to generate configuration information for a web service, see [ServiceModel Metadata Utility Tool &#40;Svcutil.exe&#41;](https://msdn.microsoft.com/library/aa347733.aspx).For a full description of the static type parameters for the WSDL type provider, see [WsdlService Type Provider](https://msdn.microsoft.com/en-us/visualfsharpdocs/conceptual/wsdlservice-type-provider-%5bfsharp%5d).
+<br />  There are two ways to specify configuration settings for a WSDL connection, by using the app.config file for the project, or by using the static type parameters in the type provider declaration. You can use svcutil.exe to generate appropriate configuration file elements. For more information about using svcutil.exe to generate configuration information for a web service, see [ServiceModel Metadata Utility Tool &#40;Svcutil.exe&#41;](https://msdn.microsoft.com/library/aa347733.aspx). For a full description of the static type parameters for the WSDL type provider, see [WsdlService Type Provider](https://msdn.microsoft.com/visualfsharpdocs/conceptual/wsdlservice-type-provider-%5bfsharp%5d).
 <br />
```

Comment:
@cartermp I've noticed that we use some HTML elements throughout the F# content. Some are needed like the one for the pipes inside tables but in some cases they seem unnecessary like the <br> tags in this topic. Should we open a low-pri issue to clean HTML tags throughout the content?